src/puppet-wechat.ts

```diff
@@ -1145,6 +1146,14 @@ export class PuppetWeChat extends PUPPET.Puppet {
 
     try {
 
+      if (rawPayload.MMStatus === MsgSendStatus.SENDING) {
+        throw new Error('message still sending')
+      }
+
+      if (rawPayload.MMStatus !== MsgSendStatus.SUCCESS) {
```

Comment:
What if the current `rawPayload` is not a message sent from local?

Could you please add some comments to document the below two cases:

1. the `rawPayload` is a message sent from others
2. the `rawPayload` is a message sent by the logged-in user.

Reply:
Can I use `sendByLocal` property to know a message sent from local or not? and why setting `sendByLocal` to false at here https://github.com/wechaty/puppet-wechat/blob/035e90461aa759b332056db9905bc5a9a4ac14e0/src/wechaty-bro.js#L426-L431
Can line 428-430 be deleted?

Reply:
I totally forgot why there has an ` m.sendByLocal = false`.

Please feel free to remove them when you can be sure that the new code can work without any problem.